tiledb/arrow/src/datatype.rs

```diff
@@ -92,6 +92,28 @@ pub fn tiledb_type_physical(
         arrow_schema::DataType::UInt64 => Some(tiledb::Datatype::UInt64),
         arrow_schema::DataType::Float32 => Some(tiledb::Datatype::Float32),
         arrow_schema::DataType::Float64 => Some(tiledb::Datatype::Float64),
+        arrow_schema::DataType::Timestamp(
```

Comment:
Not really related to the PR description but it ended up in there on the branch before I had to deal with the circular dependency.

These cases were found by PBT, as it happens.